google-cloud-storage/src/main/java/com/google/cloud/storage/Storage.java

```diff
@@ -4882,4 +5028,15 @@ default void close() throws Exception {}
   default BlobWriteSession blobWriteSession(BlobInfo blobInfo, BlobWriteOption... options) {
     return throwGrpcOnly(fmtMethodName("blobWriteSession", BlobInfo.class, BlobWriteOption.class));
   }
+
+  /**
+   * Atomically move an object from one name to another.
+   *
+   * <p>This new method is an atomic equivalent of the previous rewrite + delete, however without
```

Comment:
I think "the previous" isn't the right phrasing here, because you can still do it that way right?

Maybe this should say something more like "Using this method is an atomic equivalent of doing a {@code Storage.copy} and then {@code Storage.delete}, except that it doesn't change the metadata fields of the target object"

Reply:
I'll remove `previous`, but there are some metadata fields that are changed. I know at least generation, create_time change when the move happens.

I'll also add the cross-method links as you suggest.